upload-chunked-file.js

```diff
@@ -0,0 +1,43 @@
+function chunkFileUpload(file, serverUrl, chunkSize) {
```

Comment:
The current implementation does not validate `file`, `serverUrl`, or `chunkSize`. Add input validation to ensure these are of the expected types and values to prevent runtime errors.